CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Changelog
 
+### 0.18.2
+
+**Commit Delta**: [Change from 0.18.1 release](https://github.com/plus3it/watchmaker/compare/0.18.1...0.18.2)
+
+**Released**: 2020.04.02
+
+**Summary**:
+
+*   Updates the vault-auth-formula
```

Comment:
The changelog entry for a submodule update is:

```
*   vault-auth-formula
    * <description of change>
```

Reply:
Fixed